packages/react-google-maps-api/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+  "compilerOptions": {
+    "outDir": "./lib/",
+    "sourceMap": true,
+    "noImplicitAny": false,
```

Comment:
Too many `any` in overall? :) Wondering if that could be improved, haven't checked all cases yet.

Reply:
My main use case for any is in:
https://github.com/JustFly1984/react-google-maps-api/blob/typescript/packages/react-google-maps-api/src/utils/helper.ts

The functions there receive a component, it's props, it's updaterMap object and it's eventMap object. These are different for each component.

Reply:
Well, in that case, it might be better to ignore per case instead of turning that guard off everywhere and possibly missing some cases.